frontend/src/pages/collections/[collection]/resources/add.astro

```diff
@@ -28,28 +30,41 @@ const { collection, error } = await getCollection(collectionId || '', Astro.requ
   <div class="govuk-grid-row">
     <div class="govuk-grid-column-full">
 
-      <h1 class="govuk-heading-l">Upload file(s)</h1>
+      <h1 class="govuk-heading-l">Upload</h1>
```

Comment:
💯.  We should change the button that links to this page too (currently "Add file(s)")